backend/src/main/kotlin/org/loculus/backend/service/submission/SequenceEntriesTable.kt

```diff
@@ -44,6 +44,14 @@ object SequenceEntriesTable : Table(SEQUENCE_ENTRIES_TABLE_NAME) {
         )
     }
 
+    fun distinctOrganisms() = SequenceEntriesTable
+        .select(SequenceEntriesTable.organismColumn)
+        .withDistinct()
```

Comment:
Just a side note related to what we chatted about yesterday: this is a good sign that it would be nice to have a dedicated organism table.

Reply:
I agree this is a non-ideal function: go through all entries, possibly millions to just find out which organisms there are? We could do the same by looking at values.yaml - or as you suggest add an organisms table.

If we do that, we could then make all organism columns across all tables foreign keys into that organism table.

Reply:
Yep, always room for improvement!